File: _FACUL/LP/LP_Go/num_primos.go
package main

import (
	"fmt"
)

var fim, resto int

func main() {
	fim = 1000
	for num := 0; num < fim; num++ {
		if num == 1 {
			continue
		} else {
			if num%2 != 0 || num == 2 {
				for div := 2; div < num; div++ {
					resto = num % div
					if resto == 0 {
						break
					}
				}
				if resto != 0 || num == 2 {
					fmt.Printf("%d\t", num)
				}
			}
		}
	}
}

File: _FACUL/AED_I/AED_I_Python/num_primos.py
inicio = 0
fim = 1000
for num in range(inicio, fim+1):
  if num == 1:
    continue
  else:
    resto = num % 2
    if resto != 0 or num == 2:
      for div in range(2, num):
        resto = num % div
        if resto == 0:
          break
      if resto != 0 or num == 2:
        print(num, end='\t')